pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1514,4 +1514,28 @@ self: super: {
     };
   };
 
+  # hasn‘t bumped upper bounds
+  which = doJailbreak super.which;
```

Comment:
Oh, also, could you either create an issue upstream about this, or send a PR bumping bounds for them?

Reply:
https://github.com/obsidiansystems/which/pull/6